azure/v14.0.0/README.md

```diff
@@ -0,0 +1,92 @@
+# :zap: Giant Swarm Release v14.0.0 for Azure :zap:
+
+This is the first tenant cluster release to support Kubernetes 1.19 on Azure.
+
+Thanks for the [added support by `Azure`](https://azure.microsoft.com/en-us/updates/germany-west-central-availability-zones-now-generally-available/),
+with this release `Availability Zones` are supported for all `Workload Clusters` running in the `Germany West Central`
+region as well.
+
+Starting from this release, Azure `Workload Clusters` include by default a new application named `azure-scheduled-events`
+that leverages the [Azure scheduled events](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/scheduled-events)
+feature to automatically drain a `Kubernetes` node when the underlying virtual instance is about to be terminated.
+This makes the cluster behaviour better when the `cluster autoscaler` scales down a node pool.
```

Comment:
how does it make it better? draining is faster?

Reply:
much faster

Reply:
This ensures the workload running on the node is handled gracefully when the `cluster autoscaler` scales down a node pool.

Maybe this?

Reply:
much better, updated, thx